internal/daemon/controller/handlers/targets/credentials.go

```diff
@@ -162,13 +163,27 @@ func staticToWorkerCredential(ctx context.Context, cred credential.Static) (sess
 			},
 		}
 
+	case *credstatic.KvCredential:
```

Comment:
KV can never be a worker credential, we should drop this `case` and return an error in `default`